.genx/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@genesislcap/blank-app-seed-config",
-  "description": "Genesis Blank App Seed Configuration",
+  "packageDescription": "Genesis Blank App Seed Configuration",
```

Comment:
It'll need to stay as description, as that's the standardised package.json field name